src/app/course/courses-shared/course-authors/course-authors.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroupDirective } from '@angular/forms';
+
+@Component({
+  selector: 'app-course-authors',
+  templateUrl: './course-authors.component.html',
+  styleUrls: ['./course-authors.component.less']
+})
+export class CourseAuthorsComponent implements OnInit {
+  authors: FormControl;
+
+  constructor(private formGroupDirective: FormGroupDirective) {}
+
+  ngOnInit() {
```

Comment:
another approach is to pass the form group as Input into child components
and child component append FormControl or FormGroup to the passed group